templates/Procfile

```diff
@@ -1,2 +1,2 @@
-web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
+web: bundle exec rails server Puma -p ${PORT:-3000} -c ./config/puma.rb
```

Comment:
I don't think the rackup file is supposed to know about the server:

```
# This file is used by Rack-based servers to start the application.

require ::File.expand_path('../config/environment',  __FILE__)
run CodeTriage::Application
```

What's the problem with manually using the `puma` command in the `web:` line of the `Procfile`?